meson.build

```diff
@@ -532,6 +532,16 @@ You can disable the Wayland xkbcli programs with -Denable-wayland=false.''')
                    install_dir: dir_libexec)
         install_man('tools/xkbcli-list.1')
     endif
+
+    if get_option('enable-xkb-check-messages')
+        executable('xkb-check-messages',
+                   'tools/check-messages.c',
+                   'src/messages.h',
+                   'src/messages.c',
+                   dependencies: [tools_dep],
+                   include_directories: [include_directories('src', 'include')],
+                   install: false)
+    endif
```

Comment:
remove the option and build it unconditionally - this doesn't introduce any new dependencies (unless I'm misreading something here) and since it doesn't get installed there's no point hiding it behind an option. Apologies if my rant against new tools was unclear regarding that :)

Reply:
Done